c/default.nix

```diff
@@ -0,0 +1,13 @@
+{ stdenv, gnumake }:
+stdenv.mkDerivation {
+  name = "hello";
+
+  src = ./.;
+  nativeBuildInputs = [ gnumake ];
```

Comment:
```suggestion
```
`make` is in stdenv, so i'm not sure we need this? also if you were to switch to meson/cmake, this would need to be replaced with one of these instead
```nix
nativeBuildInputs = [ cmake ];
```
```nix
nativeBuildInputs = [ meson ninja ];
```